editor/src/clj/editor/welcome.clj

```diff
@@ -412,7 +412,10 @@
               {:graphic (make-recent-project-entry recent-project recent-projects-list prefs localization)}))
           (.setOnMouseClicked (ui/event-handler event
                                 (when (= 2 (.getClickCount ^MouseEvent event))
-                                  (open-selected-project!)))))))
+                                  (open-selected-project!))))
+          (.setOnKeyPressed (ui/event-handler event
+                      (when (= javafx.scene.input.KeyCode/ENTER (.getCode ^KeyEvent event))
+                        (open-selected-project!)))))))
```

Comment:
Please keep the code formatted. Here `(when ...)` should be indented more than `(ui/event-handler ...)` since it's nested inside it (this will break my parinfer setup...)